NetKAN/MaritimePack.netkan

```diff
@@ -0,0 +1,15 @@
+{
+    "spec_version": 1,
+    "license": "BSD",
```

Comment:
From glancing as the `License.txt` in the distribution it looks like the `BSD-3-clause`.